pkg/network/protocols/postgres/telemetry.go

```diff
@@ -88,16 +152,22 @@ func (t *Telemetry) getBucketIndex(querySize int) int {
 func (t *Telemetry) Count(tx *ebpf.EbpfEvent, eventWrapper *EventWrapper) {
 	querySize := int(tx.Tx.Original_query_size)
 
-	bucketIndex := t.getBucketIndex(querySize)
-	if bucketIndex >= 0 && bucketIndex < len(t.queryLengthBuckets) {
-		t.queryLengthBuckets[bucketIndex].Add(1)
-	}
-
+	var fulfilled counterEnum = fulfillTableAndOperation
 	if eventWrapper.Operation() == UnknownOP {
 		t.failedOperationExtraction.Add(1)
+		fulfilled = fulfillOperationNotFound
 	}
 	if eventWrapper.TableName() == "UNKNOWN" {
 		t.failedTableNameExtraction.Add(1)
+		if fulfilled == fulfillOperationNotFound {
+			fulfilled = fulfillTableAndOpNotFound
+		} else {
+			fulfilled = fulfillTableNameNotFound
+		}
+	}
+	bucketIndex := t.getBucketIndex(querySize)
+	if bucketIndex >= 0 && bucketIndex < len(t.queryLengthBuckets) {
+		t.queryLengthBuckets[bucketIndex].add(1, fulfilled)
```

Comment:
if the only usage of `add` is `add(1, fulfiled)` then let's change it to `inc(fulfilled)`

Reply:
changed

Reply:
Looks like it is still relevant